plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -30,6 +30,11 @@ import {
 } from '../scaffolder';
 import { CatalogEntityClient } from '../lib/catalog';
 import { validate, ValidatorResult } from 'jsonschema';
+import NodeGit from 'nodegit';
+
+// Enables core.longpaths on windows to prevent crashing when checking out repos with long foldernames and/or deep nesting
+// @ts-ignore
+NodeGit.Libgit2.opts(28, 1);
```

Comment:
Let's do this to fix the failing tests. (As Ben suggested above)

```ts
NodeGit?.Libgit2?.opts(28, 1);
```

Somehow jest is not able to locate these type definitions. I had tried doing `import * as Nodegit from 'nodegit'` based on this [Stackoverflow Q&A](https://stackoverflow.com/questions/50427196/jest-typeerror-cannot-read-property-of-undefined), but then the error changes and it's not able to find `.opts`.

Reply:
Thanks, @OrkoHunter ;)